macro/src/expansion.rs

```diff
@@ -68,44 +68,14 @@ fn gen_field(path: String, ident_str: String, leaf: Field, out: &mut Vec<Item>)
         ),
     };
 
+    // TODO: leaf_stmts is always empty... for now. Are there plans for this? Otherwise the code can be simplified even further!
```

Comment:
I'm pretty sure this is just left over from an earlier attempt to implement this, so we can remote it!.

Reply:
Good to know, then I'll just remove that variable.